reference/6/Microsoft.PowerShell.Management/Add-Content.md

```diff
@@ -221,6 +221,11 @@ The acceptable values for this parameter are as follows:
 - **UTF32**: Encodes in UTF-32 format.
 - **Unknown**: The encoding type is unknown or invalid; the data can be treated as binary.
 
+Beginning with PowerShell 6.2, the **Encoding** parameter also allows numeric IDs of registered code
+pages (like `-Encoding 1251`) or string names of registered code pages (like
+`-Encoding "windows-1251"`). For more information, see the .NET documentation for
+[Encoding.CodePage](https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.codepage?view=netcore-2.2).
```

Comment:
Fix the link to remove FQDN and localization.  This looks true in all the files in this PR.

Reply:
fixed